sdk/src/main/java/com/qonversion/android/sdk/dto/QProductType.kt

```diff
@@ -0,0 +1,28 @@
+package com.qonversion.android.sdk.dto
+
+import com.android.billingclient.api.BillingClient
+
+enum class QProductType(val type: Int) {
+    trial(0),
+    subscription(1),
+    inApp(2);
+
```

Comment:
Coding Conventions https://kotlinlang.org/docs/reference/coding-conventions.html#property-names :
For enum constants, it's OK to use either uppercase underscore-separated names (enum class Color { RED, GREEN }) or regular camel-case names starting with an uppercase first letter, depending on the usage.